src/App.js

```diff
@@ -1,230 +1,117 @@
+            right
+        </Link>
         </div>
-      </section>
-      <section className="grey section">
-        <div className="card grid-holder">
-          <img src={depth} alt="Pixel matrices with depth of 3" className="grid-img"/>
-          <div className="side-text">
-            <p>In order to apply our filter to each channel of an <em>RGB</em> (<em>R</em>ed, <em>G</em>reen, <em>B</em>lue) image,
-            we need our filter to be 3-dimensional as well&mdash;that is, our filter should have width, height, and depth.
-            </p>
-            <p>The width, height and depth of our filter together determine the <em>size</em> of our filter (the dimensions ). In CNNs analyzing RGB images, 3 x 3 x 3 (width x height x depth) filters are some of the most commonly-used filters,
-            along with 5 x 5 x 3 filters.</p>
-          </div>
+      )
+    }
+    else if (curPage === 5) {
+      return (
+        <div className="navigation">
+          <Link to={leftStr} className="navArrow" onClick={() => setPage(curPage - 1)}>
+            left
+        </Link>
         </div>
-      </section>
-      <FilterSizes />
-      <section className="grey section">
-        <div className="card">
-          <h1>What do CNNs look like?</h1>
-          <p>
-            Now that we know how to apply filters, let's take a look at how one layer of a CNN might use two feature-detecting filters to create feature maps.
-          </p>
+      )
+    }
+    else {
+      return (
+        <div className="navigation">
+          <Link to={leftStr} className="navArrow" onClick={() => setPage(curPage - 1)}>
+            left
+        </Link>
+          <Link to={rightStr} className="navArrow" onClick={() => setPage(curPage + 1)}>
+            right
+        </Link>
         </div>
-      </section>
-      <section className="blue section">
-        <div className="card">
-        <button id="fullscreen" onClick={handle.enter}>
-          View full image
-        </button>
+      )
+    }
 
-        <FullScreen handle={handle}>
-          <div className="cnnHolder">
-              <img src={visualizer} id="cnn-vis" alt="two filters applied to one layer of a CNN"/>
-          </div>
-        </FullScreen>
+  }
 
+  return (
+    <div className="App">
+      <Router>
+        <ScrollToTop />
+        <div class="index">
+          <Link to="/1" className="navButton" onClick={() => setPage(1)}>
+            Page 1
+          </Link>
+          <Link to="/2" className="navButton" onClick={() => setPage(2)}>
+            Page 2
+          </Link>
+          <Link to="/3" className="navButton" onClick={() => setPage(3)}>
+            Page 3
+          </Link>
+          <Link to="/4" className="navButton" onClick={() => setPage(4)}>
+            Page 4
+          </Link>
+          <Link to="/5" className="navButton" onClick={() => setPage(5)}>
+            Page 5
+          </Link>
         </div>
```

Comment:
If you'd like, you could generate this with a `.map` over `1...5` - should be less code dup :)

Reply:
I think this is the only relevant review now!